src/Duende.AccessTokenManagement/ClientCredentialsTokenManagementBuilder.cs

```diff
@@ -31,6 +31,13 @@ public ClientCredentialsTokenManagementBuilder(IServiceCollection services)
     public ClientCredentialsTokenManagementBuilder AddClient(string name, Action<ClientCredentialsClient> configureOptions)
     {
         _services.Configure(name, configureOptions);
+        _services.PostConfigure<ClientCredentialsClient>(name, client =>
+        {
+            if (string.IsNullOrEmpty(client.ClientId))

```

Comment:
Are there other parameters that are always required that we should add validation for? E.g., we could throw if there was no credential (secret or assertion) supplied.